api/candlepin-api-spec.yaml

```diff
@@ -263,6 +400,40 @@ components:
             value:
               type: string
 
+    DistributorVersionDTO:
+      description: A DTO representation of the DistributorVersion
+      allOf:
+        - $ref: '#/components/schemas/TimestampedEntity'
+        - type: object
+          required:
+            - displayName
+            - id
+            - name
+          properties:
+            id:
+              type: string
+            name:
+              type: string
+            displayName:
+              type: string
+            capabilities:
+              type: array
+              uniqueItems: true
```

Comment:
To some extent, maybe this is an area where we should evaluate our logic and clean it up ourselves. However, that aside, we could always use the container typing as a short-term (heh) patch to deal with this.

It would be nice if we could stay completely within the bounds of the OpenAPI spec, but this could be another shortcoming.